admin/model_advanced_cluster_description.go

```diff
@@ -22,6 +22,8 @@ type AdvancedClusterDescription struct {
 	CreateDate *time.Time `json:"createDate,omitempty"`
 	// Storage capacity that the host's root volume possesses expressed in gigabytes. Increase this number to add capacity. MongoDB Cloud requires this parameter if you set **replicationSpecs**. If you specify a disk size below the minimum (10 GB), this parameter defaults to the minimum disk size value. Storage charge calculations depend on whether you choose the default value or a custom value.  The maximum value for disk storage cannot exceed 50 times the maximum RAM for the selected cluster. If you require more storage space, consider upgrading your cluster to a higher tier.
 	DiskSizeGB *float64 `json:"diskSizeGB,omitempty"`
+	// Disk warming mode selection.
+	DiskWarmingMode *string `json:"diskWarmingMode,omitempty"`
```

Comment:
I understand the if we don't use it,  nil means use the default value,  as it's happening now?

Reply:
See: https://github.com/mongodb/atlas-sdk-go/blob/main/docs/doc_5_best_practices.md#working-with-pointers
Let me know if documentation is clear. We are keen to improve it.